package.json

```
@@ -36,7 +36,8 @@
     "ts-contracts": "yarn ts-contracts:web3 && yarn ts-contracts:truffle",
     "ts-contracts:truffle": "typechain --target truffle-v5 --outDir build/truffle-typings \"build/contracts/*.json\"",
     "ts-contracts:web3": "typechain --target web3-v1 --outDir build/types \"build/contracts/{BatchExchange,BatchExchangeViewer}.json\"",
-    "ganache": "ganache-cli -h 0.0.0.0 --gasLimit 8e6 --deterministic"
+    "ganache": "ganache-cli --host 0.0.0.0 --gasLimit 8e6 --deterministic",
+    "truffle exec": "sh -c 'yarn truffle exec build/common/${0%.*}.js $@'"
```

```suggestion
    "truffle-exec": "sh -c 'npx truffle exec build/common/${0%.*}.js $@'"
```

When I tried it, it looks like having a space in the didn't work :disappointed:. Also wouldn't this script call itself recursively forever the way its currently setup?